torchtune/utils/constants.py

```diff
@@ -22,3 +22,5 @@
 # total number of epochs for training; resumed training runs for
 # (total_epochs - epochs_run) number of epochs
 TOTAL_EPOCHS_KEY = "total_epochs"
+# value head key
+VALUE_HEAD_KEY = "value_head"
```

Comment:
Can you add a little more context for where this value_head is relevant?

Reply:
Going to remove this for now.